api/namex/resources/name_requests/name_request.py

```diff
@@ -265,10 +265,7 @@ def handle_patch_actions(action, model):
                 return jsonify(response_data), 200
 
             # Add the list of valid Name Request actions for the given state to the response
-            if (nr_action == NameRequestPatchActions.REQUEST_REFUND.value):
-                response_data['actions'] = []
-            else:
-                response_data['actions'] = nr_svc.current_state_actions
+            response_data['actions'] = nr_svc.current_state_actions
```

Comment:
have you tested this with the UI yet? I see you added the receipt acton for if they have a receipt of the refund, but do all the other actions still get hidden after they've done a refund?

Reply:
Yes I tested it. I can see only receipt now

Reply:
The enum contain only RECEIPT